spec/features/merchants/show_spec.rb

```diff
@@ -27,6 +27,11 @@
     @invoice_item_4 = InvoiceItem.create!(invoice_id:"#{@invoice_4.id}", item_id:"#{@item_1.id}", status: 2, quantity:1, unit_price:600)
     @invoice_item_5 = InvoiceItem.create!(invoice_id:"#{@invoice_5.id}", item_id:"#{@item_1.id}", status: 2, quantity:1, unit_price:600)
```

Comment:
I dont know if this would make a difference in your test but InvoiceItems usually will have the same invoice id but different item_ids and then that ticket will get sent to transactions for charge with that single invoice_id

Reply:
We discussed this on a zoom call - thanks, Darren for clarifying. Because the favorite customers method is just based on transactions, we decided the test doesn't need to be changed for now. But I am going to look into relocating this method to the Customer class before we merge!